minari/utils.py

```diff
@@ -197,6 +239,11 @@ def create_dataset_from_buffers(
         author (Optional[str], optional): author that generated the dataset. Defaults to None.
         author_email (Optional[str], optional): email of the author that generated the dataset. Defaults to None.
         code_permalink (Optional[str], optional): link to relevant code used to generate the dataset. Defaults to None.
+        ref_min_score( Optional[float], optional): minimum reference score from the average returns of a random policy. This value is later used to normalize a score with :meth:`minari.get_normalized_score`. If default None the value will be estimated with a default random policy.
```

Comment:
should we even compute min score if max score is not provided? we won't be able to use it anyway

Reply:
This is already being done. First checking if an `expert_policy` or `ref_max_score` is passed and then computing the respective attributes. I'll update the docs, I agree it's not very clear that functionality